k8s/ithc/common/aac/manage-case-assignment.yaml

```diff
@@ -0,0 +1,32 @@
+
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: manage-case-assignment
+  namespace: aac
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:pr-86-*
+spec:
+  releaseName: manage-case-assignment
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/aac-manage-case-assignment
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/aac/manage-case-assignment:pr-86-d67f51e0
```

Comment:
i think we can use latest prod image now. https://github.com/hmcts/cnp-flux-config/blob/aac-ithc/k8s/perftest/common/aac/manage-case-assignment.yaml

Reply:
done. thanks!